app/src/main/java/org/lineageos/selfie/ui/GridView.kt

```diff
@@ -43,12 +49,15 @@ class GridView(context: Context?, attributeSet: AttributeSet?) : View(context, a
             GridMode.ON_GOLDENRATIO -> 3
         }
 
-        if (size <= 0) {
+        if (size <= 0 || previewSize == null) {
```

Comment:
`val previewSize = previewSize ?: return`

Reply:
not sure if that's worth it...

Reply:
This way you can get rid of height and width variables and !! operators